packages/server/src/controllers/UsersController.js

```diff
@@ -4,7 +4,145 @@ import UsersValidator from '../validators/UsersValidator'
 
 const { User, UserAddress } = Models
 
+// temp dummy models
+let Users = { }
+let UserEmails = { }
+let UserSessions = { }
+
+
 export default {
+  async details(req, res) {
+    const { params: { address } } = req
+    let body = {
+      emailExists: false,
+      emailVerified: false,
+      addressVerified: false,
+      notificationsDisabled: false
+    }
+    Object.assign(body, Users[address])  // Object.assign acts as a merge to avoid empty keys
```

Comment:
`Object.assign()` has usually been superceded by the object-rest-spread operator instead:

```
Object.assign(body, Users[address])
```

is equivalent to the usually more preferred version:

```
const body = {
  key: 'value',
  ...Users[address],
}
```

This may be a useful time machine reference for syntax like this: https://github.com/Khan/style-guides/blob/master/style/javascript.md#dont-use-underscore

Reply:
oh nice will update